component/upgrade/patch-crds.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -eu
+
+cd /export
+
+for crd in ${CRDS_TO_PATCH}; do
+  # Export and remove all metadata properties except `metadata.name`
+  kubectl get crd "${crd}" -o json > "${crd}.json"
+  jq 'del(.status) | del(.metadata) | .metadata.name += "'${crd}'"' "${crd}.json" > "${crd}.patched.json"
+  # Apply the CRD again (this shouldn't change anything, except updating the annotation "kubectl.kubernetes.io/last-applied-configuration")
+  # You will also see some warnings in the output mentioning the annotation.
+  # This is expected and actually required.
+  kubectl apply -f "${crd}.patched.json"
```

Comment:
We may need `--as cluster-admin` for Openshift here, let's see

Reply:
It works without 😊